process/banning

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" >
+<title>Process to suspend or remove participants from groups</title>
+<link rel="stylesheet" href="/StyleSheets/generic-base-1.css"
+type="text/css">
+<link rel="stylesheet" type="text/css" href="../assets/main.css">
+<link rel="shortcut icon" href="/Icons/WWW/Literature.gif">
+<link rel="start" href="../" >
+<style>
+  .IssueLabel-big {
+    font-weight: 600;
+    padding: 4px 8px;
+    border-radius: 3px;
+    line-height: 1.5;
+    box-sizing: border-box;
+    text-decoration: none;
+  }
+</style>
+</head>
+<body>
+
+<div id="header">
+<span class="logo"><a href="/"><img src="/Icons/WWW/w3c_home_nb"
+alt="W3C" height="48" width="72"></a></span>
+    <div class="breadcrumb">
+    <a href="/Member/">Member</a> &#x2192; <a href="/Guide/">The Art of
+Consensus</a> &#x2192;
+<h1>Process to suspend or remove participants from groups</h1>
+</div>
+<p class="baseline">This <strong>Guidebook</strong> is the collected
+wisdom of the W3C Group Chairs and other collaborators.</p>
+</div>
+
+    <div class="toolbox box">
+      <h4>Nearby</h4>
+      <ul>
+        <li>
+          <a href="/Consortium/Process/">W3C Process Document</a>
+        </li>
+        <li>
+          <a href="/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>
+        </li>
+      </ul>
+    </div>
+    
+    <div id="Content">
+
+<p>The W3C process document empowers the Director to <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">suspend or remove participants from groups</a>. This document provides details about implementation and delegation.</p>
+
+<p>1. Consistent with the practices below, the Director delegates authority to suspend or remove participants:</p>
+<ul>
+<li>from a single group, to the CEO;</li>
+<li>from a single group, where the participant is not formally a member of the group, to the chair(s) of the group; and</li>
+<li>from all W3C groups, to the CEO and COO.</li>
```

Comment:
Suggest "to the CEO and COO, jointly" as until reading further, it looked as though it was to either.